packages/frontend/src/components/Editor/SlateToolbar.tsx

```diff
@@ -85,7 +87,7 @@ export const SlateToolbar = () => {
 
   return (
     <Slate editor={editor} value={value}>
-      <ToolbarPaperWrapper elevation={2}>
+      <ToolbarPaperWrapper elevation={2} aria-label="toolbar">
```

Comment:
These aria-labels should be translated texts. Maybe you can as @noravux for help?